galsim/phase_psf.py

```diff
@@ -1247,6 +1252,9 @@ class OpticalPSF(GSObject):
                             individual aberration.  Note that aberrations[1] is piston (and not
                             aberrations[0], which is unused.)  This list can be arbitrarily long to
                             handle Zernike polynomial aberrations of arbitrary order.
+    @param annular_zernike  Boolean indicating that aberrations specify the amplitudes of annular
```

Comment:
Is the intent that ```annular_zernikes``` can be used in either the case of specifying ```aberrations``` or when using the ```spher```, ```trefoil1```, etc. kwargs?  If so, does that make sense?  I thought those names were reserved for the circular zernike definition but I could be completely off base here.